docs/TESTING.md

```diff
@@ -0,0 +1,25 @@
+# Testing
+
+### E2E tests on BrowserStack
+
+#### Prerequisites
+
+-   [BrowserStack](https://www.browserstack.com) account
+
+### Running tests
+
+This will run all the tests in `showcase/src/e2e-test`
```

Comment:
I think we should elaborate a bit more here to:

1. explain why is a best practice to put all E2E tests in one folder 
2. mention that the path of this folder is used in the protractor config (in case they want to change it)